ext/ffi/lib.rs

```diff
@@ -1096,11 +1179,16 @@ where
   let cb = v8::Local::<v8::Function>::try_from(v8_value)?;
 
   let isolate: *mut v8::Isolate = &mut *scope as &mut v8::Isolate;
+  LOCAL_ISOLATE_POINTER.with(|s| s.replace(isolate));
```

Comment:
Maybe add a check that we've replaced a null pointer here?

Reply:
Added a check to only replace the null pointer once.